includes/class-wc-payments.php

```diff
@@ -935,6 +935,22 @@ function () {
 					}
 				}
 			);
+
+			// This injects the payments API into core, so the WooCommerce Blocks plugin is not necessary.
+			// The payments API is currently only available in feature builds (with flag `WC_BLOCKS_IS_FEATURE_PLUGIN`).
+			// We should remove this once it's available in core by default.
+			if ( ! defined( 'WC_BLOCKS_IS_FEATURE_PLUGIN' ) && class_exists( 'Automattic\WooCommerce\Blocks\Payments\Api' ) ) {
+				$blocks_package_container = Automattic\WooCommerce\Blocks\Package::container();
+				$blocks_package_container->register(
+					Automattic\WooCommerce\Blocks\Payments\Api::class,
+					function ( $container ) {
+						$payment_method_registry = $container->get( Automattic\WooCommerce\Blocks\Payments\PaymentMethodRegistry::class );
+						$asset_data_registry     = $container->get( Automattic\WooCommerce\Blocks\Assets\AssetDataRegistry::class );
+						return new Automattic\WooCommerce\Blocks\Payments\Api( $payment_method_registry, $asset_data_registry );
+					}
+				);
+				$blocks_package_container->get( Automattic\WooCommerce\Blocks\Payments\Api::class );
```

Comment:
Is this needed here? :thinking:

Reply:
You mean this line?

```php
$blocks_package_container->get( Automattic\WooCommerce\Blocks\Payments\Api::class );
```

It replaces [this line](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/src/Domain/Bootstrap.php#L103) from the Woo Blocks plugin, and needs to happen after [this](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/src/Domain/Bootstrap.php#L265-L272).

Reply:
Got it!